testing.rst

```diff
@@ -158,22 +158,26 @@ As an example, a test could look like this::
 
 .. tip::
 
-    To run your functional tests, the ``WebTestCase`` class bootstraps the
-    kernel of your application. In most cases, this happens automatically.
-    However, if your kernel is in a non-standard directory, you'll need
-    to modify your ``phpunit.xml.dist`` file to set the ``KERNEL_DIR``
-    environment variable to the directory of your kernel:
+    To run your functional tests, the ``WebTestCase`` class needs to know which
+    is the application kernel to bootstrap it. The kernel class is usually
+    defined in the ``KERNEL_CLASS`` environment variable (included in the
+    default ``phpunit .xml-dist`` file provided by Symfony):
 
     .. code-block:: xml
 
         <?xml version="1.0" charset="utf-8" ?>
         <phpunit>
             <php>
-                <server name="KERNEL_DIR" value="/path/to/your/app/" />
+                <!-- the value is the FQCN of the application kernel -->
+                <server name="KERNEL_CLASS" value="App\Kernel" />
```

Comment:
now should be `<env ... />`? https://github.com/symfony/recipes/blob/master/phpunit/phpunit/4.7/phpunit.xml.dist#L12